Packs/Microsoft365Defender/ReleaseNotes/1_1_2.md

```diff
@@ -0,0 +1,9 @@
+
+#### Scripts
+##### MS365DefenderUserListToTable
+- The MS365DefenderUserListToTable folder is now inside the Scripts folder.
+##### MS365DefenderCountIncidentCategories
+- The MS365DefenderCountIncidentCategories folder is now inside the Scripts folder.
```

Comment:
These are dev notes - not something visible to the user
just put `Maintenance and stability enhancements.` instead for both of these

Also put a note for the docker image updates

Reply:
Fixed